sonar-duplications/src/main/java/net/sourceforge/pmd/cpd/SourceCode.java

```diff
@@ -59,9 +62,7 @@
     protected abstract Reader getReader() throws Exception;
 
     protected List<String> load() {
-      LineNumberReader lnr = null;
-      try {
-        lnr = new LineNumberReader(getReader());
+      try (LineNumberReader lnr = new LineNumberReader(getReader())) {
```

Comment:
Indeed, Jacoco does not currently "understand" try-with-resource, it badly detect 8 branches to cover.
See https://github.com/jacoco/jacoco/issues/15 for the whole story.